src/ucp/core/ucp_request.h

```diff
@@ -283,11 +283,13 @@ struct ucp_request {
                             union {
                                 /* Used by rndv/put and rndv/put/frag */
                                 struct {
-                                    /* Which lanes need to flush (0 in fence mode) */
-                                    ucp_lane_map_t flush_map;
+                                    /* Next lane to flush is the first set bit
+                                       in rpriv->flush_map that's >= flush_lane */
+                                    ucp_lane_index_t flush_lane;
```

Comment:
> Minor: WDYT about renaming to `next_flush_lane` and `next_atp_lane`? I think it would make code more self-explained.

I thought about it but it's not exactly next lane, it's more like a lower bound on the next lane, and didn't want to make a longer name that is not so accurate anyway